stratum/hal/lib/tdi/tdi_sde_helpers.cc

```diff
@@ -394,43 +394,65 @@ ::util::Status GetAllEntries(
     std::shared_ptr<::tdi::Session> tdi_session,
     ::tdi::Target tdi_dev_target, const ::tdi::Table* table,
     std::vector<std::unique_ptr<::tdi::TableKey>>* table_keys,
-    std::vector<std::unique_ptr<::tdi::TableData>>* table_values) {
+    std::vector<std::unique_ptr<::tdi::TableData>>* table_datums) {
```

Comment:
Why change the variable name from `table_values` to `table_datums`?

"datums" isnt a good name, since the plural of "datum" is "data". Should switch back to original variable name

Reply:
I explicitly changed `table_datums` to `table_values` in an earlier edit because `datums` is grammatically incorrect. Please to not undo my correction.

`datum` is singular. `data` is plural. `datums` is gibberish.

I used `table_values` because the code already had a `table_data` and I wanted to limit the extent of the change.